sw/host/opentitanlib/src/test_utils/mem.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use std::time::Duration;
+
+use crate::io::uart::Uart;
+use crate::test_utils::e2e_command::TestCommand;
+use crate::test_utils::rpc::{UartRecv, UartSend};
+use crate::test_utils::status::Status;
+
+// Bring in the auto-generated sources.
+include!(env!("mem"));
+
+impl MemRead32Req {
```

Comment:
Would it make sense to have some constructors to make it nicer? Ie
```rust
pub fn new(addr: u32) -> MemRead32Resp {
   ...
}
```

Reply:
I think it probably makes more sense for this to be a extension trait or a standalone function rather than method on `MemRead32Req`.